src/qml/pages/main.qml

```diff
@@ -61,6 +61,6 @@ ApplicationWindow {
                     progress: nodeModel.verificationProgress
                 }
             }
-         }
+        }
```

Comment:
To minimize chances of conflicts during repo syncing, suggesting to drop this change or move it into a separated commit.

Reply:
dropped